addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json

```diff
@@ -35,6 +35,14 @@
     },
     {
       "path":".svn/entries",
+      "content":["FILE","DIR"],
+      "links":["https://www.oreilly.com/library/view/version-control-with/9780596510336/ch08s02s01.html"],
+      "type":"svn_dir",
+      "source":"snallygaster"
```

Comment:
@sgerlach Are these really both from snallygaster?

Reply:
Actually information I gathered from these resources as well as snallygaster and some local research I did
https://github.com/NetSPI/SVNentriesParser
https://cirt.net/svnpristine

Reply:
Okay, I was trying to ensure we attributed things properly. I guess the .svn/entries one is originally from snallygaster though You've improved it here, and the .svn/wc.db one is actually "ZAP Development Team" ... Does that seem logical?

Reply:
I think so yes. Does it need some re-organization...?

Reply:
I think it's just a one line tweak, I'll fire in a PR.